telemetry/definitions/commonDefinitions.json

```diff
@@ -312,7 +338,7 @@
             "name": "toolId",
             "type": "string",
             "description": "The tool being installed",
-            "allowedValues": ["session-manager-plugin", "dotnet-lambda-deploy", "dotnet-deploy-cli"]
+            "allowedValues": ["session-manager-plugin", "dotnet-lambda-deploy", "dotnet-deploy-cli", "aws-cli"]
```

Comment:
Sanity check: did you intend to add this change? It is not in the PR Description (same with lines 121-123)

Reply:
Same PR is fine IMO (though to be fair I do this myself way more than I should). Expanding the allowed values is non-breaking.

Reply:
https://github.com/aws/aws-toolkit-common/pull/318